src/java_tools/buildjar/javatests/com/google/devtools/build/java/turbine/javac/BUILD

```diff
@@ -72,10 +64,6 @@ java_test(
     data = [
         "//third_party/java/jdk/langtools:javac_jar",
     ],
-    jvm_flags = [
-        # Simulates how Bazel invokes turbine
-        "-Xbootclasspath/p:$(location //third_party/java/jdk/langtools:javac_jar)",
```

Comment:
Why did you remove these flags? They may be the reason why the buildtools tests are failing on jdk8

Reply:
Because the build fails with them on JDK11.

Reply:
OK, I think it's ok to disable the tests for JDK8 for the time being... I'll investigate the issue  on Monday. Thanks!

Reply:
Where